src/studio/nuxeo-studio/nuxeo-studio-release-notes/2018-04-02-studio-73-9.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'April 2, 2018'
+description: .
+tree_item_index: 989
+review:
+  comment: ''
+  date: '2018-04-02'
+  status: ok
+toc: true
+---
+
+{{! multiexcerpt name='summary-v73-9'}}
+You can now configure your project as a real application containing dependencies on any public addon without having to make a dedicated custom bundle for that.
+{{! /multiexcerpt}}
+
+## Declare Dependencies on any Package
+
+Nuxeo Studio will now let you declare a dependency to any public marketplace addon, instead of a select few. This removes the need to setup a development environment and create a custom bundle for the sake of having all your dependencies installed automatically.
```

Comment:
Nuxeo Studio now lets you declare dependencies to any public marketplace addon, instead of a select few. This removes the need to setup a development environment and create a custom bundle in order to install all your dependencies automatically.